conf/application.conf

```diff
@@ -217,6 +217,11 @@ agent-services {
   url = "/agent-services-account"
 }
 
+agent-client-services {
+   host = "http://localhost:9448"
```

Comment:
You will need to make an `app-config-base` PR to blank out this host in the environments